source/community/release_notes/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _Open edX Release Notes:
+
+###############################
+Open edX Platform Release Notes
+###############################
+
+The *Open edX Platform Release Notes* provide information about releases, migrations, and other changes and updates to the Open edX platform.
+
+.. Note to Editors: As we add new releases, be sure to move the older
+   un-supported release to the toctree in the old_releases.rst file.
+
+.. toctree::
+    :maxdepth: 2
+
+    Olive: The current release <olive>
```

Comment:
This seems kind of... limp? Can we emphasize it better? Can we make it like, bigger or put it in its own section like

# Current Release

The current release is Olive: `<olive>`

# Unsupported Releases

old_releases

Reply:
Since it's in a table of contents it's hard to emphasize like that, but we also have this link on the main page for the latest release and not for the others so I'm not too worried about it.

Reply:
What about "Olive: The current release" // "Previous, unsupported releases:" ?